CMakeLists.txt

```diff
@@ -379,8 +379,12 @@ else()
     message(STATUS "AES support enabled")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -maes")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -maes")
-  elseif(ARM) #NB ARMv8 DOES support AES, but not yet coded
-    message(STATUS "AES support disabled (not available on ARM)")
+  elseif(ARM6 OR ARM7)
+    message(STATUS "AES support disabled (not available on ARMv6 or ARMv7)")
+  elseif(ARM8)
+    message(STATUS "AES support enabled for ARMv8-A")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=armv8-a+crypto")
```

Comment:
I mean, just replace this line with `set(CPU_FEATURES "${CPU_FEATURES}" "crypto")`. Does my earlier post make more sense?